doc/release/migration_guide_from_1.x_to_2.0.rst

```diff
@@ -15,12 +15,12 @@ work with both NetworkX v1.x and v2.0.
 We have made some major changes to the methods in the Multi/Di/Graph classes.
 The methods changed are explained with examples below.
 
-With the release of NetworkX 2.0 we are moving to a view/iterator reporting API.
-We have moved many methods from reporting lists or dicts to iterating over
+With the release of NetworkX 2.0 we are moving to a view/iterable reporting API.
+We have changed many methods from reporting lists or dicts to iterating over
 the information. Most of the changes in this regard are in the base classes.
 Methods that used to return containers now return views (inspired from
 `dictionary views <https://docs.python.org/3/library/stdtypes.html#dict-views>`_
-in Python) and methods that returned iterators have been removed.
+in Python) and methods that returned an iterable have been removed.
```

Comment:
Why change ```iterator``` to ```iterable```?  Lots of things are iterable... A list is iterable....  
The API used to return an iterator, e.g. ```G.nodes_iter()``` similar to the old ```dict.iterkeys()```.
I think we should leave iterator in most of these places, but I didn't check carefully. Maybe it should be replaced in some of these spots.

Reply:
By that logic, you might as well say that you returned generators since that's the most specific ABC.  I just figured that most people would only expect a method like `nodes_iter()` to be iterable.  They wouldn't expect it to necessarily be an iterator.  But okay, we can revert this.